src/env.ts

```diff
@@ -98,6 +98,10 @@ const CORECLR_PROFILER = "{846F5F1C-F9AE-4B07-969E-05C26BC060D8}";
 const CORECLR_PROFILER_PATH = "/opt/datadog/Datadog.Trace.ClrProfiler.Native.so";
 const DD_DOTNET_TRACER_HOME = "/opt/datadog";
 
+// Java tracer env variables
+const JAVA_TOOL_OPTIONS_VAR = "JAVA_TOOL_OPTIONS";
+const JAVA_TOOL_OPTIONS = '-javaagent:"/opt/java/lib/dd-java-agent.jar" -XX:+TieredCompilation -XX:TieredStopAtLevel=1';
```

Comment:
I've noticed some error logs from the agent, which seems to occur because of the missing env. config setting `DD_JMXFETCH_ENABLED: false` - see [DD configure-environment-variables](https://docs.datadoghq.com/serverless/installation/java/?tab=gradle#configure-environment-variables)...

<img width="1041" alt="image" src="https://user-images.githubusercontent.com/81381/161949114-18d2b035-a5ea-4521-9581-c129deada7d7.png">

- [ ] include `DD_JMXFETCH_ENABLED: false` environment config

Reply:
@lizapressman could you maybe check also the logs from your test function and see if you also get periodically errors like this?

<img width="1038" alt="image" src="https://user-images.githubusercontent.com/81381/161951945-ed8a1014-964b-4b46-be2a-4e4b9d8d017c.png">

```sh
[dd.trace 2022-04-06 09:35:41:163 +0000] [dd-task-scheduler] ERROR datadog.communication.monitor.DDAgentStatsDConnection - Unable to create StatsD client - localhost:8125
```

This doesn't seem to be specific to the Serverless plugin, as we also get this errors when we integrate the layers and configs directly without the plugin. Also the log level doesn't seem to take effect, as we're seeing the logs unaggregated on INFO level, despite the fact that we have configured `logLevel: warn` or `DD_LOG_LEVEL: WARN`.

Our `serverless.yml` config looks like this:

```yml
...
provider:
  environment:
    DD_ENV: ${self:provider.stage}
    DD_SERVICE: ${self:service}
    DD_TRACE_SAMPLE_RATE: ${self:custom.datadogTracingSampleRate.${opt:stage, self:provider.stage}}
    DD_VERSION: ${git:sha1}
...
custom:
  datadogTracingSampleRate:
    dev: 1.0
    int: 0.8
    prod: 1.0
  ...
  datadog:
    addExtension: true
    apiKey: ${ssm:.../apiKey}
    site: datadoghq.eu

    flushMetricsToLogs: true
    addLayers: true
    logLevel: warn
    enableTags: true
    injectLogContext: true
    integrationTesting: false
```

Reply:
@lizapressman could you maybe check also the logs from your test function and see if you also get periodically errors like this?

<img width="1038" alt="image" src="https://user-images.githubusercontent.com/81381/161951945-ed8a1014-964b-4b46-be2a-4e4b9d8d017c.png">

```sh
[dd.trace 2022-04-06 09:35:41:163 +0000] [dd-task-scheduler] ERROR datadog.communication.monitor.DDAgentStatsDConnection - Unable to create StatsD client - localhost:8125
```

This doesn't seem to be specific to the Serverless plugin, as we also get this errors when we integrate the layers and configs directly without the plugin. Also the log level doesn't seem to take effect, as we're seeing the logs unaggregated on INFO level, despite the fact that we have configured `logLevel: warn` or `DD_LOG_LEVEL: WARN`.

Our `serverless.yml` config looks like this:

```yml
...
provider:
  environment:
    DD_ENV: ${self:provider.stage}
    DD_SERVICE: ${self:service}
    DD_TRACE_SAMPLE_RATE: ${self:custom.datadogTracingSampleRate.${opt:stage, self:provider.stage}}
    DD_VERSION: ${git:sha1}
...
custom:
  datadogTracingSampleRate:
    dev: 1.0
    int: 0.8
    prod: 1.0
  ...
  datadog:
    addExtension: true
    apiKey: ${ssm:.../apiKey}
    site: datadoghq.eu

    flushMetricsToLogs: true
    addLayers: true
    logLevel: warn
    enableTags: true
    injectLogContext: true
    integrationTesting: false
```

Reply:
Ok, thank, I'll do so. 👍🏻